### DISCUSSION TITLE
How to handle datasets that contain multiple resolutions?

### DISCUSSION THREAD

**Discussion by BriannaLind** [opengeos/NASA-Earth-Data#3] [Q&A] [answered]
What would be a smart way to handle this? some data sets, like this [one](https://doi.org/10.5067/ASTER/AST_07XT.003) include bands with multiple resolutions.... some 15m and some 30m

![image](https://github.com/opengeos/NASA-Earth-Data/assets/104585874/c0885767-4dea-4d6f-96c2-726455b4c229)

**Comment by giswqs** [comment]
Good point. I need to look into the metadata to see how they are recorded in the metadata.

**Comment by giswqs** [comment]
We need to standardize the way metadata is recorded. Otherwise, we will end up with different numbers of columns for different datasets

**Comment by giswqs** [comment]
This is the metadata of this dataset. It indeed has multiple resolutions recorded in it. I probably need to modify the spreadsheet to record the resolutions as a list rather than as individual columns. 

```text
     "SpatialExtent": {
       "SpatialCoverageType": "HORIZONTAL",
       "HorizontalSpatialDomain": {
         "Geometry": {
           "CoordinateSystem": "CARTESIAN",
           "BoundingRectangles": [
             {
               "NorthBoundingCoordinate": 90.0,
               "WestBoundingCoordinate": -180.0,
               "EastBoundingCoordinate": 180.0,
               "SouthBoundingCoordinate": -90.0
             }
           ]
         },
         "ResolutionAndCoordinateSystem": {
           "GeodeticModel": {
             "HorizontalDatumName": "World Geodetic System (WGS84)"
           },
           "HorizontalDataResolution": {
             "VariesResolution": "Varies",
             "GenericResolutions": [
               {
                 "XDimension": 15.0,
                 "YDimension": 15.0,
                 "Unit": "Meters"
               },
               {
                 "XDimension": 30.0,
                 "YDimension": 30.0,
                 "Unit": "Meters"
               }
             ]
           }
         }
       },
       "GranuleSpatialRepresentation": "GEODETIC"
     },
```

**Comment by giswqs** [comment, marked as answer]
The spreadsheet has been updated. It now contains multi-resolution info. 

![image](https://github.com/opengeos/NASA-Earth-Data/assets/5016453/88c65a38-7cd5-4ee2-b9d1-9ba3a3369a94)
